python/tvm/topi/x86/batch_matmul.py

```diff
@@ -47,6 +47,7 @@ def batch_matmul_vnni_compute(cfg, x, y):
             axis=ak,
         ),
         tag="batch_matmul_vnni",
+        attrs={"schedule_rule": "batch_matmul_vnni"},
```

Comment:
Im not familiar with metascheduler, what is `schedule_rule`?

Reply:
This is not related to this PR, but I needed this for testing purpose and it is too trivial to send a PR for this. It is for injecting manually-written TIR schedules in `TE compute -> create_primfunc -> Meta scheduling` flow, rather than auto-scheduling them entirely. See how I use it in https://gist.github.com/masahi/10e96c94fc3557d91b191e674068f785#file-vnni_dense_tir-py-L356-L360